.infra/index.ts

```diff
@@ -93,7 +93,13 @@ const deadLetterTopic = new Stream(digestDeadLetter, {
 createSubscriptionsFromWorkers(
   name,
   isAdhocEnv,
-  addLabelsToWorkers(workers, { app: name }),
+  addLabelsToWorkers(workers, { app: name })
```

Comment:
It's better to add a subapp here as well. Otherwise the autoscaling for this will take into account the personalized digest messages